euth/contrib/emails.py

```diff
@@ -0,0 +1,31 @@
+from email.mime.image import MIMEImage
+
+import pkg_resources
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import select_template
+from django.utils.translation import get_language
+
+
+def send_email_with_template(receivers, template, context):
+    languages = [get_language(), 'en']
+    subject = select_template(['emails/{}.{}.subject'.format(template, lang)
+                               for lang in languages])
+    plaintext = select_template(['emails/{}.{}.txt'.format(template, lang)
+                                 for lang in languages])
+    html = select_template(['emails/{}.{}.html'.format(template, lang)
+                            for lang in languages])
+
+    mail = EmailMultiAlternatives(
+        subject=subject.render(context).strip(),
+        body=plaintext.render(context),
+        from_email=settings.DEFAULT_FROM_EMAIL,
+        to=receivers,
+    )
+    mail.mixed_subtype = 'related'
+    f = pkg_resources.resource_stream('euth_wagtail', 'static/images/logo.png')
```

Comment:
You should use [`from django.contrib.staticfiles.finders.find()`](https://docs.djangoproject.com/en/1.10/ref/contrib/staticfiles/#finders-module) instead.